modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/domain/nodeCollateral.scala

```diff
@@ -0,0 +1,12 @@
+package io.constellationnetwork.dag.l0.domain
+
+import io.constellationnetwork.schema.nodeCollateral.UpdateNodeCollateral
+import io.constellationnetwork.security.signature.Signed
+
+object nodeCollateral {
+
+  sealed trait NodeCollateralOutput
```

Comment:
Same, this needs a better name, Request or Event would be more appropriate here since it represents a user submitted value, not a calculated value.